tests/infra_test.go

```diff
@@ -585,35 +589,28 @@ var _ = Describe("[Serial][sig-compute]Infrastructure", func() {
 
 			// We need a token from a service account that can view all namespaces in the cluster
 			By("extracting virt-operator sa token")
-			token, _, err := tests.ExecuteCommandOnPodV2(virtClient,
-				&op,
-				"virt-operator",
-				[]string{
-					"cat",
-					"/var/run/secrets/kubernetes.io/serviceaccount/token",
-				})
-			Expect(err).ToNot(HaveOccurred(), "failed executing command on virt-operator")
+			cmd := []string{"cat", "/var/run/secrets/kubernetes.io/serviceaccount/token"}
+			token, stderr, err := tests.ExecuteCommandOnPodV2(virtClient, &op, "virt-operator", cmd)
+			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf(remoteCmdErrMsg, strings.Join(cmd[:], ","), token, stderr, err))
```

Comment:
`strings.Join(cmd[:], ",")` Would ` ` be better?

Reply:
Why do we need to copy `cmd`?
Wouldn't `strings.Join(cmd, ",")` achieve the same affect without copying?

Reply:
@xpivarc @iholder-redhat You are correct! I've fixed both of your remarks. Thank you